docs/sdks/server-sdks/dotnet.mdx

```diff
@@ -164,7 +164,7 @@ The SDK will invoke the `LogAssignment` method with an `event` object that conta
 | `Subject` (string)                        | An identifier of the subject or user assigned to the experiment variation                                              | "ed6f85019080"                      |
 | `Variation` (string)                      | The experiment variation the subject was assigned to                                                                   | "control"                           |
 | `Timestamp` (DateTime)                    | The time when the subject was assigned to the variation                                                                | 2021-06-22T17:35:12.000Z            |
-| `SubjectAttributes` (Map<String, object>) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment method | `Map.of("device","iOS")`            |
+| `SubjectAttributes` (Map\<String, object\>) | A free-form map of metadata about the subject. These attributes are only logged if passed to the SDK assignment method | `Map.of("device","iOS")`            |
```

Comment:
I don't think DotNet has `Map`. I iamgine this should be `(IDictinary\<string, Object\>)` with the example value `\["device"\] = "iOS"`